Blog/config/database.yml

```diff
@@ -5,24 +5,25 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  adapter: mysql2
+  encoding: utf8
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: Blog_development
```

Comment:
App name and tables should be lowercase

Reply:
I knew that tables should be lower case, I will remember about the app name in the future as well.

Reply:
As for more tests, as suggested below, it could be a good idea to make a separate branch for carrying them out, so I think I should put that off till the next step.